recipes/cobra/meta.yaml

```diff
@@ -7,45 +7,50 @@ package:
 source:
   fn: cobra-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/c/cobra/cobra-{{ version }}.tar.gz
-  sha256: 104e1f73f78d463849161143e8f2b98fcd7cb5ce8ee2c5b0c704ce3153036e30
+  sha256: ee74a9d585eb3fc40b3e8b6f7516bb80bb178c8413cd071c6c3189d93337cce5
 
 build:
-  number: 1
-  skip: True # [osx]
+  number: 0
+#  skip: True # [osx]
 
 requirements:
   build:
     - python
     - setuptools
     - six
-    - cython
-    - gcc
-    - glpk
+    - future
+    - swiglpk
+    - ruamel.yaml <0.15
+    - numpy >=1.6
+    - pandas >=0.17.0
+    - optlang >=1.2.5
+    - tabulate
 
   run:
     - python
     - six
-    - numpy
-    - scipy
-    - lxml
-    - python-libsbml
-    - glpk
-    - libgcc
+    - future
+    - swiglpk
+    - ruamel.yaml <0.15
+    - numpy >=1.6
+    - pandas >=0.17.0
+    - optlang >=1.2.5
+    - tabulate
 
 test:
   imports:
     - cobra
     - cobra.core
-    - cobra.design
     - cobra.flux_analysis
     - cobra.io
     - cobra.manipulation
-    - cobra.solvers
     - cobra.test
-    - cobra.topology
+    - cobra.util
 
   requires:
```

Comment:
As we don't actually run the tests, we can remove the `test/requires` section. The import of `cobra.test` will still succeed as in `cobra/test/__init__.py` the `pytest` imports are guarded by a `try ... except`.